architecture/kubernetes_infrastructure.adoc

```diff
@@ -9,43 +9,41 @@
+== Overview
+http://kubernetes.io[Kubernetes] is a system for managing containerized applications across a set of containers or hosts, providing deployment, maintenance, and application-scaling mechanisms. Docker is used to package, instantiate, and run containerized applications.
 
-A Kubernetes cluster consists of a master and a set of nodes. 
+A Kubernetes cluster consists of a master and a set of nodes.
 
 == Master
-
-The Kubernetes master is the host or hosts that contain the master components (API Server, Controller Manger Server, etcd, etc.).
+The master is the host or hosts that contain the master components, including the API server, controller manager server, and [sysitem]#etcd#. The master manages link:#node[nodes] in its Kubernetes cluster and schedules link:kubernetes_model.html#pod[pods] to run on nodes.
 
 === API Server
+The Kubernetes API server serves the main API, and validates and configures the data for pods, services, and replication controllers. The Kubernetes API server also assigns pods to nodes and synchronizes pod information with service configuration.
 
-The Kubernetes API server serves the main API, and validates and configures the data for pods, services, and replication controllers. The Kuberbetes API server also assigns pods to nodes and synchronizes pod information with service configuration. 
-
-=== etcd
-
-Persistent master state is stored in etcd. Other components watch etcd for changes and eventually bring themselves into the desired state described in etcd.
+=== [sysitem]#etcd#
+Persistent master state is stored in [sysitem]#etcd#. Other components watch [sysitem]#etcd# for changes and eventually bring themselves into the desired state described in [sysitem]#etcd#.
 
 === Controller Manager Server
-
-The controller manager server watches etcd for changes to replication controller objects and then uses the API to enforce the desired state.
+The controller manager server watches [sysitem]#etcd# for changes to replication controller objects and then uses the API to enforce the desired state.
 
 == Node
+A node is a system providing the run-time environments for containers. Each node in a Kubernetes cluster is managed by the link:#master[master]. Nodes have the services necessary for this management and to run pods, including Docker, a link:#kubelet[kubelet], and a link:#service-proxy[service proxy].
 
-Kubernetes validates the node with health checks and ignores it until these health checks are passed. Note that Kubernetes continues to check nodes until they are valid.
+Nodes are created from your cloud provider, or from your own physical or virtual systems. Kubernetes interacts with node objects that are a representation of those nodes. The master uses the information from node objects to validate nodes with health checks. A node is ignored until the health checks are passed, however the master continues to check nodes until they are valid.
```

Comment:
[Suggestions]
Nodes are created from a cloud provider, or from a physical or virtual system. 

Information from the node objects is used by the master to validate nodes with health checks. Although a node is ignored until it passes the health checks, the master continues to check nodes until they are valid.